pkg/database/mysql.go

```diff
@@ -0,0 +1,26 @@
+package database
+
+
+type MySQLDatabase struct {
+	DB *bun.DB
+}
+
+var (
+	once sync.Once
+	dbInstance *MySQLDatabase
+)
+
+func NewMySQLDatabase(conf *entities.Config) *MySQLDatabase {
+	once.Do(func() {
+		sqlDB, err := sql.Open("mysql", "")
+		if err != nil {
+			panic(err)
+		}
+		dbInstance := bun.NewDB(sqlDB, mysqldialect.New())
+	})
+	return dbInstance
+}
+
+func (MySQLDatabase *MySQLDatabase) GetDB() *bun.DB {
+	return dbInstance.DB
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/10#issuecomment-2425499428>#d26e6c</a><span> - 10/21/2024, 09:12 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The GetDB method is not using its receiver to access the DB field. Instead, it's using a global variable dbInstance, which may lead to unexpected behavior and thread-safety issues.
</div>
<div id="fix">
<b>Fix:</b> Modify the GetDB method to use the receiver's DB field instead of the global dbInstance variable.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -24,3 +24,3 @@
  func (MySQLDatabase *MySQLDatabase) GetDB() *bun.DB {
 -	return dbInstance.DB
 +	return MySQLDatabase.DB
  }
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged